tests/common.go

```diff
@@ -3690,6 +3699,23 @@ func CreateS3Bucket(bucketName string) {
 	})
 	expect(err).NotTo(haveOccurred(),
 		fmt.Sprintf("Failed to wait for bucket [%v] to get created. Error: [%v]", bucketName, err))
+
+	if retainCount > 0 && objectLock == true {
+		// Update ObjectLockConfigureation to bucket
+		enabled := "Enabled"
+		_, err = S3Client.PutObjectLockConfiguration(&s3.PutObjectLockConfigurationInput{
+			Bucket: aws.String(bucketName),
+			ObjectLockConfiguration: &s3.ObjectLockConfiguration{
+				ObjectLockEnabled: aws.String(enabled),
+				Rule: &s3.ObjectLockRule{
+					DefaultRetention: &s3.DefaultRetention{
+							Days: aws.Int64(retainCount),
+							Mode: aws.String(objectLockMode)}}}})
+
+		expect(err).NotTo(haveOccurred(),
+		fmt.Sprintf("Failed to update Objectlock config with Retain Count [%v] and Mode [%v]. Error: [%v]", retainCount, objectLockMode, err))
```

Comment:
Sprintf just formats the string, use log to print the log messsage

Reply:
Done